### PR TITLE
Update docs to reflect support for autopilot DR

### PR DESCRIPTION
**Pull request by vishalnayak**

### CODE REVIEW

File: website/content/docs/concepts/integrated-storage/autopilot.mdx [hunk 1]
```diff
@@ -70,7 +70,10 @@ behavior. Autopilot gets initialized with the following default values.
 Performance secondary clusters have their own Autopilot configuration, managed
 independently of their primary.
 
-DR secondary clusters do not currently support Autopilot.
+DR secondary clusters will also their own Autopilot configuration (starting in
+Vault 1.8.0), managed independently of their primary. The [Autopilot
+API]((api/system/storage/raftautopilot) will be authorized via the DR Operation
```
Comment (ncabatoff) [inline website/content/docs/concepts/integrated-storage/autopilot.mdx:75]: I'm not keen on this phrasing ("will be").  How about instead: The API uses DR operation tokens for authorization.

Comment (vishalnayak) [reply website/content/docs/concepts/integrated-storage/autopilot.mdx]: Done.

File: website/content/docs/concepts/integrated-storage/autopilot.mdx [hunk 2]
```diff
@@ -70,7 +70,10 @@ behavior. Autopilot gets initialized with the following default values.
 Performance secondary clusters have their own Autopilot configuration, managed
 independently of their primary.
 
-DR secondary clusters do not currently support Autopilot.
+DR secondary clusters will also have their own Autopilot configuration (starting
+in Vault 1.8.0), managed independently of their primary. The [Autopilot
+API]((api/system/storage/raftautopilot) uses DR operation tokens for
```
Comment (ncabatoff) [inline website/content/docs/concepts/integrated-storage/autopilot.mdx:75]: Link is broken due to extra `(`: https://vault-krvsmed3q-hashicorp.vercel.app/docs/concepts/integrated-storage/autopilot

Comment (vishalnayak) [reply website/content/docs/concepts/integrated-storage/autopilot.mdx]: Oh gosh. Fixed.

File: website/content/docs/concepts/integrated-storage/autopilot.mdx [hunk 3]
```diff
@@ -70,7 +70,10 @@ behavior. Autopilot gets initialized with the following default values.
 Performance secondary clusters have their own Autopilot configuration, managed
 independently of their primary.
 
-DR secondary clusters do not currently support Autopilot.
+DR secondary clusters will also have their own Autopilot configuration (starting
+in Vault 1.8.0), managed independently of their primary. The [Autopilot
+API](api/system/storage/raftautopilot) uses DR operation tokens for
```
Comment (ncabatoff) [inline website/content/docs/concepts/integrated-storage/autopilot.mdx:75]: Link's broken, needs a leading slash I think.

**Comment by ncabatoff** [comment]
Hey Vishal, did you see my comment about the broken link?

File: website/content/docs/concepts/integrated-storage/autopilot.mdx (hunk 3 above)
Comment (vishalnayak) [reply website/content/docs/concepts/integrated-storage/autopilot.mdx]: I thought I had fixed it. This is embarrassing. Fixed.

**Review by ncabatoff — APPROVED** [review]

**Pull request merged**